### PR TITLE
Getting ready for Simmetrics 4.1.1.klara.1

### PR DESCRIPTION
**Pull request by simon-shlomo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,349 +1,282 @@
...
-						<extensions>true</extensions>
-						<configuration>
-							<serverId>ossrh</serverId>
-							<nexusUrl>https://oss.sonatype.org/</nexusUrl>
-							<autoReleaseAfterClose>false</autoReleaseAfterClose>
-						</configuration>
-					</plugin>
-					<plugin>
-						<groupId>org.apache.maven.plugins</groupId>
-						<artifactId>maven-source-plugin</artifactId>
```
Comment (thofer) [inline pom.xml:297]: The source plugin is here. Why do you remove it first, in order to re-add it again?

Using profile 'release' to build source is absolutely fine

File: pom.xml [hunk 2]
```diff
@@ -1,349 +1,282 @@
...
-								<id>attach-sources</id>
-								<goals>
-									<goal>jar-no-fork</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-					<plugin>
-						<groupId>org.apache.maven.plugins</groupId>
-						<artifactId>maven-javadoc-plugin</artifactId>
```
Comment (thofer) [inline pom.xml]: Is also great to be here

File: pom.xml [hunk 3]
```diff
@@ -1,349 +1,282 @@
...
-								<id>attach-javadocs</id>
-								<goals>
-									<goal>jar</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-					<plugin>
-						<groupId>org.apache.maven.plugins</groupId>
-						<artifactId>maven-gpg-plugin</artifactId>
```
Comment (thofer) [inline pom.xml:321]: This one is a problem. It's good to remove it

File: pom.xml [hunk 4]
```diff
@@ -1,349 +1,282 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (thofer) [inline pom.xml]: Wrong indentation

File: pom.xml [hunk 5]
```diff
@@ -1,349 +1,282 @@
...
-	<packaging>pom</packaging>
+    <groupId>com.github.mpkorstanje</groupId>
+    <artifactId>simmetrics</artifactId>
+    <version>4.1.1.klara.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
 
-	<name>SimMetrics</name>
-	<inceptionYear>2014</inceptionYear>
-	<description>A Java library of similarity and distance metrics e.g. Levenshtein distance and Cosine similarity. All
+    <name>SimMetrics</name>
```
Comment (thofer) [inline pom.xml]: I do not understand how you added source artifact.

**Review by thofer — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (simon-shlomo) [reply pom.xml:297]: then we keep it here

**Review by thofer — APPROVED** [review]

**Pull request merged**